lib/liquid/combinators/general.ex

```diff
@@ -160,6 +164,12 @@ defmodule Liquid.Combinators.General do
     |> traverse({__MODULE__, :to_atom, []})
   end
 
+  def logical_operator_coma do
+    empty()
+    |> string(",")
+    |> traverse({__MODULE__, :to_or, []})
```

Comment:
Are there any other tags that use logical comma operators or is it only in `when` tags where its supported?